test_regress/t/t_instrumentation.cpp

```diff
@@ -0,0 +1,42 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//
+// DESCRIPTION: Verilator: Verilog Test module
+//
+// This file ONLY is placed under the Creative Commons Public Domain, for
+// any use, without warranty, 2024 by Wilson Snyder.
+// SPDX-License-Identifier: CC0-1.0
+
+#include <verilated.h>
+#include <verilated_vcd_c.h>
+
+#include <fstream>
+
+#include VM_PREFIX_INCLUDE
+
+unsigned long long main_time = 0;
+double sc_time_stamp() { return main_time; }
+
+int main(int argc, char** argv) {
+    Verilated::debug(0);
+    Verilated::commandArgs(argc, argv);
+
+    std::unique_ptr<VM_PREFIX> top{new VM_PREFIX{"top"}};
+
+    std::ofstream logFile("obj_vlt/t_instrumentation/simulation_output.log");
```

Comment:
Would open a file like this:

```suggestion
    std::ofstream logFile(VL_STRINGIFY(TEST_OBJ_DIR)  "/simulation_output.log");
```

but there's no need to open anything, just print to stdout and use in the .py

`test.execute(expect_filename=test.golden_filename)
`